<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="delete_record">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_data">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>View Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>12</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="insert_record">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Record</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="c_cell">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Cellphone of person to add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="c_email">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Email of the person to add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="c_name">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Name of person to add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="c_cname">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Company name of person to add</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>654</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
